nekoyume/Assets/_Scripts/UI/Widget/Popup/ItemCountPopup.cs

```diff
@@ -36,37 +34,31 @@ protected override void Awake()
         {
             base.Awake();
 
-            if (cancelButtonText != null)
-            {
-                cancelButtonText.text = L10nManager.Localize("UI_CANCEL");
-            }
-            submitButton.SetSubmitText(L10nManager.Localize("UI_OK"));
+            submitButton.Text = L10nManager.Localize("UI_OK");
             if (informationText != null)
             {
                 informationText.text = L10nManager.Localize("UI_RETRIEVE_INFO");
             }
 
             if (cancelButton != null)
             {
-                cancelButton.OnClickAsObservable()
-                    .Subscribe(_ =>
-                    {
-                        _data?.OnClickCancel.OnNext(_data);
-                        AudioController.PlayCancel();
-                    })
-                    .AddTo(_disposablesForAwake);
-                CloseWidget = cancelButton.onClick.Invoke;
+                cancelButton.OnClick = () =>
+                {
+                    _data?.OnClickCancel.OnNext(_data);
+                    AudioController.PlayCancel();
```

Comment:
Why these `AudioContoller.PlayXxx()` invoke used or not used in same `TextButton` component's click method?

Reply:
I just intended to keep legacy `OnClick` method, but seems like this breaks the consistency. Just moved it to component's `OnClick` event in inspector.